frontend/src/bundles/common/components/input/input.tsx

```diff
@@ -15,6 +15,7 @@ type Properties<T extends FormValues> = {
     name: FieldInputProps<T>['name'];
     placeholder?: string;
     icon?: 'right' | 'none';
+    sx?: object;
```

Comment:
A better type for sx is : SystemStyleObject

Reply:
Thanks